src/core/parser/parse_fline.h

```diff
@@ -47,7 +47,10 @@
 #define SIP_VERSION_LEN 7
 
 #define HTTP_VERSION "HTTP/1."
-#define HTTP_VERSION_LEN (sizeof(HTTP_VERSION)-1)
+#define HTTP_VERSION_LEN (sizeof(HTTP2_VERSION)-1)
```

Comment:
Was the above change somehow done unintentionally? Because I see no reason to have `HTTP_VERSION_LEN` be based on size of `HTTP*2*_VERSION` value.

Reply:
Yes, it was not done intentionally. I'll fix and force push if you agree.